calico/security/encrypt-cluster-pod-traffic.md

```diff
@@ -186,7 +211,7 @@ With the above command, {{site.prodname}} will not encrypt any of the pod traffi
 To enable encryption for pod traffic on node `my-node` again:
 
 ```bash
-calicoctl patch felixconfiguration node.my-node --type='merge' -p '{"spec":{"wireguardEnabled":true}}'
+calicoctl patch felixconfiguration node.my-node --type='merge' -p '{"spec":{"wireguardEnabled":true,"wireguardEnabledV6":true}}'
```

Comment:
I think instead of just providing a command to turn IPv4 and IPv6 wireguard on and off here, we should probably refer users to turn on wireguard only for the IP version

Reply:
Good point! My intention was that this section was to serve as an example (for dual stack in this case), but I can see if that's not clear. I still think we might want to provide commands (to make it clear that we are patching the node-specific FelixConfig), so I could add the commands from v4-only, v6-only and v4+v6, directing the user to use whichever is appropriate? Or do you think that could be too confusing?